features/cubic-bezier-easing.yml

```diff
@@ -0,0 +1,6 @@
+name: cubic-bezier() easing
+description: The `cubic-bezier()` easing function interpolates along a smooth curve, creating animations and transitions with continuous changes in speed. The `ease`, `ease-in`, `ease-out`, and `ease-in-out` CSS properties are presets for common scenarios.
+spec: https://drafts.csswg.org/css-easing-2/#cubic-bezier-easing-functions
+group: css
```

Comment:
Should this go in the `Transitions (CSS)` group?

Reply:
Probably both the `Transitions (CSS)` and `Animations (CSS)` groups

Reply:
Updated the groups to include both `transitions` and `animation`, but this is now blocked pending GH-1919, where the `transitions` group will be finalized